### ISSUE TITLE
Which role is required?

### ISSUE THREAD

**Issue by hosebei** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#3802]
Can you please outline, which role is required to manage App policies? At the begininng of the article, it is stated "As an admin", but what Admin?
I have tried with Teams Service Administrator, but this does not allow me to manage the App Policies in the teams admin portal.
thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d51bb7b1-801e-6801-dcaf-afa3e38e6941
* Version Independent ID: d8e726a1-cbf2-53c7-da4d-cc155b48612b
* Content: [Manage custom app policies and settings in Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/teams-custom-app-policies-and-settings#feedback)
* Content Source: [Teams/teams-custom-app-policies-and-settings.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/teams-custom-app-policies-and-settings.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by hosebei** [comment]
as addition, the following message appears on top of the page:
We can't load the apps catalog data right now. Please try again. If it's still a problem, contact {SupportLink}.
----
And yes, the support link is not redacted or something, the message that appears is shown exactly like I have posted it.

**Comment by scanum** [comment]
@hosebei Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by get-itips** [comment]
Hello @hosebei 
I could reproduce what you are reporting here, I would say that this might be only available for Global Admins, @LanaChin can you confirm so we can adjust the doc? Thank you

**Comment by hosebei** [comment]
hi @scanum do you have any update on this topic? This is kind of a showstopper to give Global Admin permissions to our Teams Administration Team, just to handle App policies. If the requirement of Global Admin persisit, we need to build a process around that, because we can not assign GA role to them.

**Comment by get-itips** [comment]
Hello @hosebei how are you?
Here we are limited to Documentation changes, as we are waiting for the author's confirmation to update the article, I would also kindly suggest you to visit https://microsoftteams.uservoice.com/ to submit this as an idea so the Product Group receives the feedback.
Hope this info helps you.

**Comment by hosebei** [comment]
hi @get-itips fine and you?
yes I know about uservoice, thanks for the suggestion. But first I need the confirmation, that GA is required to do such workloads as App Policies. Maybe there is another option instead of GA, which is currently not outlined in the docs, as it states "As an admin".

**Comment by get-itips** [comment]
@LanaChin Lana, can you shed some light here? Thanks

**Comment by LanaChin** [comment]
Looking into this, thanks.

**Comment by LanaChin** [comment]
Hi @hosebei - We got confirmation that users who have the Teams service administrator role can manage app policies. We've updated the app policies documentation to clarify that global admins and Teams service admins can manage app policies in the Teams admin center.  Thanks for helping to improve our documentation. If your issue persists, the best way forward is to open a service request in your tenant. Here's info on that: https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online.
